src/data/contributors/sneha1827

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@sneha1827, The file-name is wrong. It should be a `json`.  Just rename it to `sneha1827.json`, else is great.